version/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+set(llvm_source_dir ${LLVM_DIR})
```

Comment:
Do I need to put copyright or some sort of license?

I see from other CMakeLists.txt files, they have `SPDX-License-Identifier: Apache-2.0`, should I use the same one here?